rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/PolicyResource.java

```diff
@@ -112,6 +119,11 @@ public Status getStatus(String application, String entityToken, String policyId)
     public Response start( String application, String entityToken, String policyId) {
         Policy policy = brooklyn().getPolicy(application, entityToken, policyId);
 
+        if (!Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.START_POLICY, policy)) {
+            throw WebResourceUtils.forbidden("User '%s' is not authorized to remove policies",
```

Comment:
Message should say `User '%s' is not authorized to start the policy '%s'`
Please also check the parameters for the templated message